app/assets/stylesheets/_summary-card.scss

```diff
@@ -0,0 +1,29 @@
+.summary-card {
+  margin-bottom: 20px;
+  padding: 15px;
+  width: 100%;
+  @include box-shadow(0 1px 4px rgba(0,0,0,.3));
+
+  .sc-row {
+    margin-bottom: 10px;
+  }
+
+  .sc-row-last {}
```

Comment:
Building upon the BEM comment above, this could change to: `&__row--last`, since the block is the `summary-card`, the element is the `row`, and `last` is the modifier.